src/App.js

```diff
@@ -1,29 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
+import './components/Calculator.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```

Comment:
- [ ] Kindly note that according to the project requirements, you are required to use Class based components in all of your components. This as well should be changed from stateless to stateful components. 

![image](https://user-images.githubusercontent.com/86318284/183977027-6e0e36e9-aef2-4048-af76-08e62d70d48f.png)
 

Below I have included for you how it is supposed to be:

```
// eslint-disable-next-line react/prefer-stateless-function
class App extends Component {
  render() {
    return (<Calculator />);
  }
}
```